src/kt_variant.h

```diff
@@ -137,66 +137,89 @@ namespace ktvariant {
         des->increment_position(encode_float(src_color.a, des->get_cursor()));
     }
 
-    template <class T>
-    static void to_kvariant_fromNATIVECORETYPE(SharedBuffer* des, const Variant& src) {
-        des->increment_position(
-                encode_uint64(reinterpret_cast<uintptr_t>(memnew(T(src.operator T()))), des->get_cursor())
-        );
-    }
-
     static void to_kvariant_fromDICTIONARY(SharedBuffer* des, const Variant& src) {
         set_variant_type(des, Variant::Type::DICTIONARY);
-        to_kvariant_fromNATIVECORETYPE<Dictionary>(des, src);
+        des->increment_position(
```

Comment:
Why is this needed? Did the template we had earlier not work?

Reply:
Not on Linux, this was failing on `operator T()`